config/presets/fastnet.go

```diff
@@ -61,6 +61,8 @@ func fastnet() config.Config {
 	conf.SMESHING.Opts.Throttle = true
 	// Override proof of work flags to use light mode (less memory intensive)
 	conf.SMESHING.ProvingOpts.Flags = postCfg.RecommendedPowFlags()
+	conf.SMESHING.ProvingOpts.Nonces = 16
+	conf.SMESHING.ProvingOpts.Threads = 1
```

Comment:
These 2 shouldn't need to be set. Fastnet uses `config.DefaultConfig` as base, not `config.MainnetConfig()`.